android_splash_screen/lib/main.dart

```diff
@@ -42,10 +42,10 @@ class MyHomePage extends StatefulWidget {
   final String title;
 
   @override
-  _MyHomePageState createState() => _MyHomePageState();
+  MyHomePageState createState() => MyHomePageState();
```

Comment:
Change to `State<MyHomePage> createState() => _MyHomePageState();`

Reply:
Updated